prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml

```
@@ -2349,6 +2350,7 @@ data:
           "dashLength": 10,
           "dashes": false,
           "datasource": "prometheus-meta",
+          "description": "This dashboard indicates how many time series each instance holds",
```

```suggestion
          "description": "This graph indicates how many time series each instance holds",
```

This is a little opinionated but maybe we can completely omit the words "This dashboard"/"This graph" because one will be able to see the description by hovering over the `i` tag on that particular grafana panel only. In some cases maybe having this will make sense :) 

Same for the other cases.